docroot/themes/humsci/humsci_basic/src/scss/README.md

```diff
@@ -0,0 +1,14 @@
+# Theming
+
+The Humsci Basic theme provides the styles for a variety of different themes.
+
+## Adding a color palette for a theme
+1. Create a new settings partial with color pairings and globals SCSS maps
+  - Use a theme specific prefix for SCSS maps and utility classes
+  - Define a default color pairing variable
+1. Define the custom properties for the theme `:root` in `elements/_base.scss` partial
+  - Duplicate an existing set of custom properties
```

Comment:
now with your mixin they dont have to do this right?

Reply:
Oh, I thought I deleted that line 🤔